src/components/localization/language-dropdown.tsx

```diff
@@ -63,7 +63,7 @@ const Absolute = styled.div<AbsoluteProps>`
         }
     }};
     left: -7rem;
-    max-height: 90vh;
+    max-height: 70vh;
```

Comment:
Hi @niloo-fs , instead of making this 70vh for all devices, I think let's do this in media query small devices like iPhone SE

Reply:
Sure, will change this